js/file-upload.js

```diff
@@ -90,53 +90,100 @@ jQuery(function($) {
     });
 
     // Deleting File
-    $(".ppom-wrapper").on('click', '.u_i_c_tools_del', function(e) {
+    document.querySelector('.ppom-wrapper').addEventListener('click', async function(e) {
```

Comment:
We can confirm to make sure the selector exists before adding an event listener.